### PR TITLE
Add license file

### LINKED ISSUE #32: Add a license?
I'd love an explicit `LICENSE` file please! :)

### PR DESCRIPTION
**Pull request by alexpricedev**
Closes #32

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) Mark Wubben <mark@novemberborn.net> (novemberborn.net)
```
Comment (novemberborn) [inline LICENSE]: Would you mind changing this?

```suggestion
Copyright (c) Mark Wubben (https://novemberborn.net)
```

**Review by novemberborn — COMMENTED** [review]
Thanks!

I don't think it needs my email address. I tried to edit your PR but I don't seem able to push the changes. Not sure why. Could you have a go?

**Comment by alexpricedev** [comment]
Np! I'll squash my commits. 1 sec

**Comment by novemberborn** [comment]
Thanks again!

**Pull request merged**